kornia/geometry/transform/crop.py

```diff
@@ -73,18 +75,21 @@ def crop_and_resize(tensor: torch.Tensor, boxes: torch.Tensor, size: Tuple[int,
         [0, dst_h - 1],
     ]], device=tensor.device).expand(points_src.shape[0], -1, -1)
 
-    return crop_by_boxes(tensor, points_src, points_dst, interpolation)
+    return crop_by_boxes(tensor, points_src, points_dst, interpolation, align_corners)
 
 
-def center_crop(tensor: torch.Tensor, size: Tuple[int, int], interpolation: str = 'bilinear') -> torch.Tensor:
+def center_crop(tensor: torch.Tensor, size: Tuple[int, int],
+                interpolation: str = 'bilinear',
+                align_corners: bool = True) -> torch.Tensor:
     r"""Crops the given tensor at the center.
 
     Args:
         tensor (torch.Tensor): the input tensor with shape (C, H, W) or
           (B, C, H, W).
         size (Tuple[int, int]): a tuple with the expected height and width
           of the output patch.
-
+        align_corners (bool): mode for grid_generation. Default: False. See
```

Comment:
you set in the signature to True and docs says False. Which is the correct one ?

Reply:
True is correct and docs were copied for everywhere because of vast amount. Thanks for catch, I'll do PR with docs fixes